cryptlex/cryptlex-enterprise/values.yaml

```diff
@@ -250,6 +250,8 @@ webApi:
   # Postmark
   postmark:
     accountToken: ""
+  googleAnalytics:
+    key: "G-XXXXXXX"
```

Comment:
@mumin-khan set deafult value to an empty string, sample value cam be added as comment. We can also club google related settings under google (recaptacha and analytics)